kani-driver/src/call_cbmc.rs

```diff
@@ -51,6 +54,7 @@ impl KaniSession {
                 // the best possible fix is port to rust instead of using python, or getting more
                 // feedback than just exit status (or using a particular magic exit code?)
             }
+            println!("Verification Time: {}s", elapsed);
```

Comment:
In case of non-termination, this would line would never be reached right?

Reply:
Yes, and also in case CBMC exited with an error.